packages/tools/src/tools/annotation/splines/types/CardinalSplineProps.ts

```diff
@@ -0,0 +1,11 @@
+import { SplineProps } from './SplineProps';
+
+/**
+ * Cardinal spline properties
+ */
+export type CardinalSplineProps = SplineProps & {
+  /** Scale that must be in 0-1 range */
+  scale?: number;
+  /** Fixed scale used by children classes (Catmull-Rom and Linear splines) */
+  fixedScale?: boolean;
+};
```

Comment:
after the review is done, before merging, can you move these to types folder?

Reply:
Moved